cronjobs/ftp_connection.py

```diff
@@ -0,0 +1,88 @@
+# Python 3 module
+# -*- coding: utf-8 -*-
+
```

Comment:
I think we never have a blank line here.

Reply:
done